_posts/en/newsletters/2020-08-19-newsletter.md

```diff
@@ -0,0 +1,208 @@
+    was believed to be easier to implement for existing software that
+    created public keys (see Newsletters [#83][news83 tiebreaker] and
+    [#96][news96 bip340 update]).
+
+    This week, Pieter Wuille [posted][wuille tiebreaker] to the
+    Bitcoin-Dev mailing list a tentative proposal to update BIP340 again
+    so that both public keys and signatures would use the evenness
+    algorithm.  It was previously thought that the squared version would
+    be faster to compute during signature verification and so would help
+    speed up full nodes, but a [recent PR][libsecp256k1 #767] to
+    libsecp256k1 by Peter Dettman with a significant performance
+    improvement caused developers to question this belief and revealed
+    that an earlier benchmark that favored the squared version wasn't
+    actually providing a fair comparison between the two tiebreaking
+    methods.  It now seems that both methods should be about equal in
+    performance.
+
+    Several respondents replied that, even though they already
+    implemented code using both tiebreaker methods, they'd prefer to see
+    the specification changed to use just the evenness method.  Anyone
+    else with an opinion is encouraged to respond to the mailing list
+    thread.
+
+- **Proposed BIP for P2P protocol feature negotiation:** Suhas Daftuar
+  [posted][daftuar negotiation] a suggestion to create a [separate
+  BIP][bip-negotiation] for the P2P feature negotiation method that
+  became a part of [BIP339][] (see [Newsletter #87][news87
+  negotiation]).  The protocol change is simple: when a node receives a
+  P2P `version` message from a newly connected peer, the node should
+  ignore any messages it doesn't understand until a `verack` message has
+  been received from that peer.  Those messages, like the new BIP339
+  `wtxidrelay` message, may signal what features the remote peer
+  supports.
+
+    This behavior is already implemented in an unreleased development
+    version of Bitcoin Core.  If the maintainers of other
+    implementations, or anyone else, opposes this change, please respond
+    to the mailing list thread.
+
+## Changes to services and client software
+
+*In this monthly feature, we highlight interesting updates to Bitcoin
+wallets and services.*
+
+- **Crypto Garage announces P2P Derivatives beta application on Bitcoin:**
+  In a [blog post][cg p2p derivatives blog], Crypto Garage outlines a beta
+  application for conducting P2P [DLC-based][dlcs] derivatives on Bitcoin regtest
+  and testnet. The application allows the specification of a financial
+  agreement and creation of a corresponding locking transaction of funds between
+  the two parties of that agreement. Upon contract maturity, a price oracle
+  provides a signature for the closing transaction spending the amounts
+  corresponding to agreement.
+
+- **Specter Desktop adds batching:**
+  [Specter Desktop][specter github], the hardware wallet multisig-focused GUI
```

Comment:
```suggestion
  [Specter Desktop][specter github], a hardware wallet multisig-focused GUI
```

I think Sjors Provoost and others are working on allowing Bitcoin Core's native GUI to access HWI, while Andrew Chow's native descriptor wallet work is making multisig easier, so at some point Bitcoin Core GUI will arguably also be a "hardware wallet multisig-focused GUI".  To prepare for that possible outcome, maybe say "a" rather than "the" here.